target/linux/ramips/dts/ApexeraWiBot.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-wibot", "ralink,mt7620a-soc";
+	model = "Apexera WiBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "wibot:blue:wan";
+			gpios = <&gpio1 5 1>;
+		};
+
+		wifi {
+			label = "wibot:blue:wifi";
+			gpios = <&gpio1 4 1>;
+		};
+
+		nand {
+			label = "wibot:blue:nand";
+			gpios = <&gpio1 3 1>;
+		};
+
+		boot {
+			label = "wibot:blue:boot";
+			gpios = <&gpio1 11 1>;
+			default-state = "on";
+		};
+
+		reset {
+			label = "wibot:blue:reset";
+			gpios = <&gpio2 23 1>;
+		};
+
+		ctrl3g {
+			label = "wibot:blue:ctrl3g";
+			gpios = <&gpio2 21 1>;
+		};
+
+		usb0 {
+			label = "wibot:blue:usb0";
+			gpios = <&gpio2 20 1>;
+			default-state = "on";
+		};
+
+		usb1 {
+			label = "wibot:blue:usb1";
+			gpios = <&gpio2 22 1>;
+		};
+        };
+};
+
+&gpio0 {
+	status = "okay";			
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&uart {
+	status = "okay";
+};
+
+&i2c {
+	status = "okay";
+
+	pcf8563: rtc@51 {
+		status = "okay";
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	m25p80@0 {
+		#address-cells = <1>;
```

Comment:
drop the address and size cells here. They are not required and will cause a dtc compiler warning.